package.json

```diff
@@ -21,6 +23,9 @@
         "@tailwindcss/forms": "^0.5.2",
         "@tailwindcss/line-clamp": "^0.4.0",
         "@tailwindcss/typography": "^0.5.0",
+        "@testing-library/dom": "^9.0.0",
+        "@testing-library/jest-dom": "^5.16.5",
+        "@testing-library/user-event": "^14.4.3",
```

Comment:
These are all testing deps for poking and prodding at DOM elements. `jsdom` lets us thinly simulate a document headlessly in vitest.